src/components/Select.vue

```diff
@@ -887,6 +897,9 @@
        * @return {array}
        */
       filteredOptions() {
+        if (this.filterOptions === false) {
```

Comment:
@pegiadise Can you add a few test cases for this? I think it'll break the tag functionality because that's actually taken care of in line 911?

Reply:
Well I was expecting the same but I actually wrote a test case for it and it did not break...